src/aiokatcp/client.py

```diff
@@ -224,7 +224,7 @@ async def handle_message(self, conn: connection.Connection, msg: core.Message) -
                 elif msg.mtype == core.Message.Type.INFORM:
                     req.informs.append(msg)
                 else:
-                    self.logger.warning("Unknown message type %s", msg.mtype)  # pragma: no cover
```

Comment:
I mean this one looks as though we can probably explicitly test for it but it would be a bit contrived.

Reply:
Sure it's possible, but also outside the scope of this PR.